tests/deseq2/test_outlier.py

```diff
@@ -105,3 +106,21 @@ def test_outlier_filtering_small_counts(self):
         res = dds.results()
         self.assertFalse(np.isnan(res["pvalue"].iloc[0]))
         self.assertTrue(np.all(np.isnan(res.pvalue[1:2])))
+
+    def test_CRXXX(self):
+        """TODO"""
+        dds = DESeqDataSet(
+            np.array(
+                [
+                    [527, 524, 10, 1541, 341, 396, 73, 31, 382, 122],
+                    [585, 498, 7, 1329, 285, 352, 58, 24, 332, 154],
+                    [33, 439, 9, 2114, 199, 115, 47, 0, 425, 19],
+                    [34, 652, 20, 3040, 308, 153, 62, 2, 543, 34],
+                    [42, 752, 14, 3796, 349, 215, 67, 1, 658, 30],
+                    [38, 565, 15, 2743, 266, 118, 58, 5, 512, 28],
+                ]
+            ),
+            clinicalData=pd.DataFrame({"group": [0, 0, 1, 1, 1, 1]}),
+            design="~ group",
+        )
+        DESeq(dds)
```

Comment:
The test is not testing anything, does it ?

Reply:
The test checks that no exception is raised during computation, but it does not check the results against a reference -- that's the job of other tests

Reply:
ok